panel/src/components/Views/Files/FileView.vue

```diff
@@ -19,20 +19,28 @@
 
 			<template #buttons>
 				<k-view-buttons :buttons="buttons" @action="onAction" />
-				<k-form-buttons />
+				<k-form-buttons @discard="onDiscard" @submit="onSubmit" />
 			</template>
 		</k-header>
 
-		<k-file-preview v-bind="preview" />
+		<k-file-preview
+			v-bind="preview"
+			:content="content"
+			@input="onInput"
+			@submit="onSubmit"
```

Comment:
`Question:` Should a preview component really submit?

Reply:
It's not needed so far and we could probably not implement it yet and only add it if it's really needed. But I thought that the previews really just could act like another section and have all the capabilities to interact with content if needed.